### PR TITLE
gitlab-ci-multi-runner: update 1.1.0 docker binary

### PR DESCRIPTION
**Pull request by timhowes**
The url for the prebuilt docker should be updated too.

### CODE REVIEW

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by nijikon** [comment]
I think that we should bump revision here.

**Comment by timhowes** [comment]
I added "revision 1" to the bottle block. Is that what you mean?

File: Library/Formula/gitlab-ci-multi-runner.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ class GitlabCiMultiRunner < Formula
 
   bottle do
     cellar :any_skip_relocation
+    revision 1
```
Comment (dunn) [inline Library/Formula/gitlab-ci-multi-runner.rb]: Sorry, it should be between the `url` and `head`.

**Comment by timhowes** [comment]
OK, I corrected it so it's in the first stanza after "url".

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**